src/Angor/Client/Pages/Wallet.razor

```diff
@@ -16,23 +7,28 @@
 @inject ILogger<Wallet> Logger;
 @inject IWalletOperations _walletOperations
 @inject IClipboardService _clipboardService
+@inject ICurrencyService _currencyService
 @inject IDerivationOperations _derivationOperations
 @inject NavMenuState NavMenuState
 @inject IEncryptionService _encryptionService
 @inject IClipboardService ClipboardService
-
-
+@using System.Text.Json
+@using Angor.Client.Storage
```

Comment:
why move the `using` under the `injected` types? out convention is that they are above, please fix this.

Reply:
fixes